test/test_mpas.py

```diff
@@ -0,0 +1,62 @@
+from uxarray._mpas import _read_mpas
+import uxarray as ux
+import xarray as xr
+from unittest import TestCase
+import numpy as np
+
+import os
+from pathlib import Path
+
+current_path = Path(os.path.dirname(os.path.realpath(__file__)))
+
+
+class TestMPAS(TestCase):
+
+    mpas_small_grid_path = current_path / 'meshfiles' / "mpas" / "x1" / 'x1.40962.static.nc'
+    mpas_large_grid_path = current_path / 'meshfiles' / "mpas" / "x1" / 'x1.655362.grid_subset.nc'
```

Comment:
These examples are pretty big files.  In my experience, having multi MB files in a repo gets cumbersome.  I would suggest something much coarser.  As examples, we have this very coarse mesh in MPS-Tools:
https://github.com/MPAS-Dev/MPAS-Tools/blob/master/mesh_tools/mesh_conversion_tools/test/mesh.QU.1920km.151026.nc

Reply:
I agree we should avoid large binary files in a git repo.